test_matrix.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from matrix import Matrix
+
+
+class TestMatrix(unittest.TestCase):
```

Comment:
add some negative tests which check that your class raises errors when needed.
_i.e: an exception should raise if the items in the tuple are `str` or `bool`_